bgpd/bgp_attr_evpn.c

```diff
@@ -245,6 +245,35 @@ void bgp_attr_evpn_na_flag(struct attr *attr, uint8_t *router_flag)
 	}
 }
 
+/*
+ * Fetch and return tunnel type if present.
+ */
+bool bgp_attr_extcom_tunnel_type(struct attr *attr, uint16_t *tunnel_type)
+{
```

Comment:
This duplicates rfapiGetTunnelType in rfapi/rfapi_import,c 
it would probably be best to (re)move the function  from rfapi and move it to bgp_attr and use it in both locations.

Reply:
rfapiGetTunnelType has 2 differences from bgp_attr_extcom_tunnel_type.
1. rfapiGetTunnelType returns BGP_ENCAP_TYPE_MPLS by default, whereas, bgp_attr_extcom_tunnel_type does not return any default value if tunnel type is not found. This is not an issue, because we can work around it by setting a default tunnel_type before calling bgp_attr_extcom_tunnel_type
2. rfapiGetTunnelType returns ENOENT (nonzero) if tunneltype is not found, and 0 if found.
bgp_attr_extcom_tunnel_type returns false (zero) if tunneltype is not found, and true (non-zero), if found. None of the callers use the return value. 

Do you think its okay to remove rfapiGetTunnelType? Or, should we keep both?

Reply:
I think a single common function would be best...

Reply:
I think this should be moved to bgp_attr.c

Reply:
@louberger Done.